ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml

```diff
@@ -524,14 +524,15 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy-private
-- as: gcp-ipi-to-multiarch-f14
+- as: gcp-ipi-to-multiarch-f28
```

Comment:
Once frequency get changed, need to re-generate cron item accordingly by running `ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh`

Reply:
good to know. done.